webroot/adm/default-keysets.html

```diff
@@ -0,0 +1,92 @@
+<html>
+<head>
+  <script src="https://ajax.googleapis.com/ajax/libs/jquery/2.1.1/jquery.min.js"></script>
+  <script src="/js/main.js"></script>
+</head>
+<body>
+<h1>UID2 Admin - Default Keyset Access Management (Sharer)</h1>
+
+<a href="/">Back</a>
+
+<br>
+<br>
+
+<h3>Inputs</h3>
+
+<label for="siteId">Site Id:</label>
+<input type="text" id="siteId" name="siteId">
+<label for="addSiteIds">Add Site Ids:</label>
+<input type="text" id="addSiteIds" name="addSiteIds">
+<label for="removeSiteIds">Remove Site Ids:</label>
+<input type="text" id="removeSiteIds" name="removeSiteIds">
+
+<br>
+<br>
+
+<h3>Operations</h3>
+
+<ul>
+  <li class="ro-shr" style="display: none"><a href="#" id="doListAll">List All</a></li>
+  <li class="ro-shr" style="display: none"><a href="#" id="doList">List</a></li>
+  <li class="ro-shr" style="display: none"><a href="#" id="doReset">Reset</a></li>
+  <li class="ro-shr" style="display: none"><a href="#" id="doUpdate">Update</a></li>
+</ul>
+
+<br>
+
+<h3>Output</h3>
+
+<div id="output">
+  <pre id="errorOutput"></pre>
+  <pre id="standardOutput"></pre>
+</div>
+
+<script language="JavaScript">
+  $(document).ready(function () {
+    $('#doListAll').on('click', function () {
+      doApiCall('GET', '/api/sharing/lists', '#standardOutput', '#errorOutput');
```

Comment:
can you use the /api/sharing/keyset endpoints?

Reply:
have keysets.html for the keyset endpoints and default-keysets.html for the list endpoints. Since they have almost the same functionality but different permissions I think we need to have different pages for them. If we don't need a UI for the list endpoints though I can get rid of that page